chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -820,12 +820,14 @@ sync.error.sslConnectionError 		= SSL connection error
 sync.error.checkConnection 			= Error connecting to server. Check your Internet connection.
 sync.error.emptyResponseServer 		= Empty response from server.
 sync.error.invalidCharsFilename		= The filename '%S' contains invalid characters.\n\nRename the file and try again. If you rename the file via the OS, you will need to relink it in Zotero.
+sync.error.apiKeyInvalid			= We could not authenticate your account with Zotero servers. Please re-enter your account details.
```

Comment:
"%S could not authenticate your account. Please re-enter your account details."

With `Zotero.appName`